modules/ROOT/pages/rate-limiting-sla-policy.adoc

```diff
@@ -0,0 +1,218 @@
+= Rate Limiting SLA-Based
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+:keywords: rate limiting SLA, api gateway, gateway, policy
+:page-aliases: rate-limiting-and-throttling-sla-based-policies.adoc
+
+[%autowidth.spread,cols="a,a"]
+|===
+>s| Policy name | Rate-limiting SLA
+>s| Summary      | Monitors access to an API by defining the maximum number of requests processed within a timespan, based on SLAs
+>s| Category | Quality of Service
+>s| First Mule version available | v4.1.0
+.4+>.^s| Returned Status Codes
+|400 - Invalid client credentials provided or quota exceeded by WSDL APIs that use SOAP v1.2
+|401 - Request blocked due to invalid client credentials (client ID or client secret) provided
+|429 - Quota exceeded; requests blocked until the current window finishes
+|500 - Invalid client credentials provided or quota exceeded by WSDL APIs that use SOAP v1.1
+
+|===
+
+The Rate-Limiting Service Level Agreement (SLA) policy enables you to control incoming traffic to an API by limiting the number of requests that the API can receive within a given timespan. When the limit is reached before the time expires, the policy rejects all requests, thereby avoiding any additional load on the backend API.
+
+To apply the Rate Limiting SLA policy to an API, you must first create a contract between the API and a registered client application. The number of requests that an API can receive within a given time is defined in the xref:manage-client-app-taks.adoc[contracts] section in API Manager. 
+
+Each request must be identified by a client ID and an optional client secret (depending on the policy configuration). To review how to obtain the client credentials of a registered client application, see xref:access-client-app-id-task.adoc[Obtaining Client Credentials of a Registered Application]. 
+
+Additionally, you can additionally configure the policy to run in a Mule runtime engine (Mule) cluster or add identifiers to define groups of requests. In a clusterized policy configuration, the quota is shared among all nodes in the cluster. For more information about how these options work, see <<examples,Examples>>.
```

Comment:
Could you please remove this part: _"or add identifiers to define groups of requests"_. Just reviewed the original document and it had that too, my bad!

Reply:
Done! Thanks!